IM_calculation/IM/im_calculation.py

```diff
@@ -310,10 +309,10 @@ def compute_measures_multiprocess(
             array_params.append(
                 (
                     waveform,
-                    ims,
-                    components_to_store,
+                    sorted(ims),
```

Comment:
Should probably sort it asap in the call chain so that way we never have unsorted IMs.